content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -116,29 +113,27 @@ sensuctl event list
 sensu-go-sandbox   keepalive   Keepalive last sent from sensu-go-sandbox at 2019-01-24 21:29:06 +0000 UTC        0   false      2019-01-24 21:29:06 +0000 UTC 
```

Comment:
Here again, on second and subsequent workthroughs, this list includes `check_disk_usage` and `curl_timings` in addition to the keepalive event.

Reply:
See https://github.com/sensu/sensu-docs/pull/1833#discussion_r336226957, let's destroy the VM at the end of the lesson. I don't believe it was ever designed to be ran again with the current VM resource.